apps/plea/models.py

```diff
@@ -484,8 +484,10 @@ def get_court(self, urn, ou_code=None):
 
         if ou_code:
             try:
-                return OUCode.objects.get(ou_code=ou_code[:5]).court
```

Comment:
I thought the OU code was two digits in positions 4 and 5, which would mean the slice would be [3:5] but I guess the OU code in this context includes the region code. Perhaps we should have another term for referring to the combined region/ou code (court identifier perhaps?) rather than overloading the term OU?